src/modules/peek/Peek.UI/Strings/en-us/Resources.resw

```diff
@@ -157,28 +157,28 @@
     <value>{0} byte</value>
     <comment>Abbreviation for the size unit byte.</comment>
   </data>
-  <data name="ReadableString_KiloByteAbbreviationFormat" xml:space="preserve">
-    <value>{0} KB</value>
+  <data name="ReadableString_KibiByteAbbreviationFormat" xml:space="preserve">
```

Comment:
I do not agree with change.  Please revert

Reply:
@Deepak-Sangle , let's please revert this change. Even thought it would be more correct, every software I provided as a sample that shows MiB numbers still presents them as "MB", so we should follow conventions as well. My bad for suggesting the change.

Reply:
Done with the changes, Peek now again show something like below image

<img src="https://user-images.githubusercontent.com/78836762/265678620-b50ff0fd-8514-465c-a23a-730cef98d7f6.png" width="600"></img>

Reply:
It should be consistent with file explorer's calculated value and unit style.  In FE, it would be MB, not MiB.  It is what people expect

When I look at the image from the bug, one question that came to mind was this just a simple rounding issue as .76 would round up to .8.

![image](https://github.com/microsoft/PowerToys/assets/1462282/b14e3542-6fe5-47bb-9834-3fa8e726cecf)

I am ok with having bytes added like FE has

Reply:
@crutkas
It's a known bug of Explorer.exe that it shows wrong information: Calculation of MiB and shown as MB.

This bug exists since ever. But I think this is something that completely confuses the users if MS will fix it.

Reply:
Then that should be a user option to override but as of now this should match FE in terms of values and units being displayed otherwise it will confuse users. Let’s fix this up and add in the setting then in a different PR to let the user pick